org.gabmus.giara.json

```diff
@@ -0,0 +1,608 @@
+                "/lib/python*/site-packages/easy-install.pth",
+                "/lib/python*/site-packages/setuptools.pth",
+                "/app/lib/python*/site-packages/easy-install.pth",
+                "/app/lib/python*/site-packages/setuptools.pth"
+            ],
+            "build-commands": [
+                "python3 setup.py install --prefix=/app --root=/"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/pallets/flask/archive/1.1.2.tar.gz",
+                    "sha256": "6da812f0fb52ad1ccfb5b3868b1dd20e420cc6e77379c2a4ecbaccd3eceb8923"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "sed -i 's/if not platform_/if not dirs/' setup.py"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "x265",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "subdir": "source",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DENABLE_CLI=OFF"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/videolan/x265",
+                    "tag": "3.4"
+                }
+            ]
+        },
+        {
+            "name": "x264",
+            "config-opts": [
+                  "--disable-cli",
+                  "--enable-shared"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://code.videolan.org/videolan/x264/-/archive/1771b556ee45207f8711744ccbd5d42a3949b14c/x264-1771b556ee45207f8711744ccbd5d42a3949b14c.tar.bz2",
+                    "sha256": "b31c5db5337873b9ee42b8cbc60b56d60ce368fa7e4c4eff90a729b36eeb8326"
+                }
+            ]
+        },
+        {
+          "name": "ffmpeg",
+          "config-opts": [
+              "--enable-rpath",
+              "--enable-gpl",
+              "--disable-static",
+              "--enable-shared",
+              "--disable-doc",
+              "--disable-alsa",
+              "--enable-libfontconfig",
+              "--enable-libfreetype",
+              "--enable-libopus",
+              "--enable-libpulse",
+              "--enable-librsvg",
+              "--enable-libvpx",
+              "--enable-libxml2",
+              "--enable-libx264",
+              "--enable-libx265",
+              "--enable-libmp3lame"
+          ],
+          "cleanup": [
+              "/share/ffmpeg/examples"
+          ],
+          "sources": [
+              {
+                  "type": "archive",
+                  "url": "https://www.ffmpeg.org/releases/ffmpeg-4.3.1.tar.xz",
+                  "sha256": "ad009240d46e307b4e03a213a0f49c11b650e445b1f8be0dda2a9212b34d2ffb"
+              }
+          ]
+        },
```

Comment:
use ffmpeg-full?

Reply:
To my understanding I cannot use more than one runtime, am I right?

Reply:
I asked on the IRC channel, solved